Build 7.18.0 fails with typefest error

Using 7.18.0 with nestjs and typescript yields:

node_modules/contentful-management/dist/typings/entities/api-key.d.ts:166:25 - error TS2307: Cannot find module 'type-fest/source/simplify' or its corresponding type declarations.

166         payload: import("type-fest/source/simplify").Simplify<Pick<Pick<import("./app-definition").AppDefinitionProps, "name" | "src" | "locations" | "parameters">, "name" | "parameters"> & Partial<Pick<Pick<import("./app-definition").AppDefinitionProps, "name" | "src" | "locations" | "parameters">, "src" | "locations">>>;

--- a/package.json
+++ b/package.json
@@ -60,7 +60,7 @@
     "contentful-sdk-core": "^6.8.0",
     "fast-copy": "^2.1.0",
     "lodash.isplainobject": "^4.0.6",
-    "type-fest": "1.0.2"
+    "type-fest": "1.1.3"
   },
   "devDependencies": {
     "@babel/cli": "^7.12.7",

